storage-node/packages/discovery/publish.js

```diff
@@ -33,56 +32,62 @@ function encodeServiceInfo(info) {
   })
 }
 
```

Comment:
Could you copy comments from the deleted code to annotate the class itself?

Reply:
done in d607a24f8eb7591013a5aae44a506fe63268bafa